meta-mender-tegra/recipes-bsp/uefi/edk2-firmware-tegra_%.bbappend

```diff
@@ -0,0 +1,5 @@
+#inherit dos2unix
```

Comment:
Nit: Any reason to include this commented out code?

Reply:
No. Will remove this. Thanks for noticing!